### DISCUSSION TITLE
Multiuser scenario

### DISCUSSION THREAD

**Discussion by lockered** [amborle/featmap#40] [General]
At the moment I am using featmap as a single user application so for now the following question is not a problem: I think I have seen most parts of the sources, although this does not mean, I already understand all parts. I am missing fragments to update the view of a project, if some other user have changed things in parallel. Is there code to update a view when other users change something?

**Comment by amborle** [comment]
I am not exactly sure what you mean. Can you elaborate? What is the use case?

**Comment by lockered** [comment]
The usecase would be a simple multiuser scenario. One person looks at a map, and another user add a feature.  I have not seen a mechanism, that updates the view of the first user automatically. But maybe I am only looking at the wrong plane.

**Comment by amborle** [comment]
Ok, what you want is realtime updates in the board. You are right, this is
not supported right now. For now, users need to reload the board to see
updates from other users.

Den sön 21 mars 2021 20:40Stefan Günther ***@***.***> skrev:
